x-pack/plugin/inference/src/main/java/org/elasticsearch/xpack/inference/external/response/openai/OpenAiErrorResponseEntity.java

```diff
@@ -60,7 +60,7 @@ public static OpenAiErrorResponseEntity fromResponse(HttpResult response) {
                     return new OpenAiErrorResponseEntity(message);
                 }
             }
-        } catch (IOException e) {
+        } catch (Exception e) {
```

Comment:
We probably want to catch all exceptions in case we have a casting issue or something